polling/check_active_db_nodes.py

```diff
@@ -65,6 +65,19 @@ def main():
   
   lockserver_handle = lockserver.create_lockserver_handle()
 
+  # Beginning of part of fix for Issue #152
+  # Perform django setup (required in Django 1.7+ (possibly 1.6.?+)) to make
+  # db/models available. We need to do this if we are using a django version
+  # that has django.setup() (1.7+?) This should happen before the repy patching
+  # of builtins, which likely occurs in the add_dy_support call in the next line
+  # of code. django.setup() expects to be able to use the builtin keyword
+  # 'type', which is patched by repy, so we should do this here, before that
+  # patching.
```

Comment:
Hmmm, this script doesn't seem to import `repyportability`/`safe`/`repy`directly  at all, and it doesn't attempt to restore an overridden `type` either. Thus, there's no point in keeping that comment block.